cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/macros/financial-item-flyout.html

```diff
@@ -26,19 +26,20 @@
 
    ========================================================================== #}
 
-{% macro expandable(value) %}
-<div class="o-expandable
-            o-expandable__padded">
-    <button class="o-expandable_header
+{% macro flyout(value) %}
+<div class="o-costs-group" data-js-hook="behavior_flyout-menu">
+    <button class="o-costs-group_header
+                   o-costs-group_target
+                   o-costs-group_target__collapsed
                    financial-item
-                   financial-item__expandable
-                   {{ ('financial-item__' + value.status | safe ) if value.status else ''}}"
+                   financial-item__flyout
+                   {{ ('financial-item__' + value.status | safe ) if value.status else ''}}" data-js-hook="behavior_flyout-menu_trigger"
```

Comment:
The data JS hook being added [here](https://github.com/cfpb/consumerfinance.gov/pull/7686/files#diff-9df3aa6bcb4547f45882fd3e86e56c0c5cbbf268451efe80abf415ef3effe6c4R37-R40), so hardcoding it into the markup is not necessary/redundant.